ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -473,14 +581,17 @@ const NetworksForm = ({
             }
           }
 
-          errorKey = 'endpointReturnedDifferentChainId';
-          errorMessage = t('endpointReturnedDifferentChainId', [
-            endpointChainId.length <= 12
-              ? endpointChainId
-              : `${endpointChainId.slice(0, 9)}...`,
-          ]);
+          if (!networkMenuRedesign) {
+            errorKey = 'endpointReturnedDifferentChainId';
+            errorMessage = t('endpointReturnedDifferentChainId', [
+              endpointChainId.length <= 12
+                ? endpointChainId
+                : `${endpointChainId.slice(0, 9)}...`,
```

Comment:
I know this is old code but since we are at it, would be nice to fix the magic numbers here as well